src/includes/Exports-Include.hs

```diff
@@ -128,6 +165,8 @@
   , hyperlinkWithParamsCode
 
     -- * Changing the title
+    -- | These code sequences are not part of ECMA-48 standard, but covered
+    -- by @xterm@ extensions. They are not very portable, especially on Windows.
```

Comment:
Windows does support these functions in its native terminals, and in emulation, so is not a good example for exceptions to portability.